scripts/third_party_versions.sh

```diff
@@ -5,7 +5,7 @@
 #
 
 # IMPORTANT: Be sure to update third_party.sha256
-ZLIB_VERSION=1.2.11
+ZLIB_VERSION=1.2.12
```

Comment:
The associated sha needs to be changed in ./scripts/third_party.sha256

Reply:
Sorry, I didn't realize that you were watching me. I was double checking that the SHA check would fail on CI, just because it's custom code and I couldn't remember testing it in a long time.

I'm going to be updating the libarchive version shortly too.

Reply:
Sorry, I was trying to sort out how to test it as well. This projects circleci setup is well done, it's nice to have it test the different architectures and versions instead of trying to iterate through those locally!